basics/getting-started/public-cloud-examples/azure-quickstart.md

```diff
@@ -91,11 +91,11 @@ az aks create --resource-group ${AKS_RESOURCE_GROUP} \
               --node-count 3
 ```
 
-Once the command succeeds, it's ready to be used.
+Once the command succeeds, the cluster is ready to be used.
 
 ## **5. Connect to an existing cluster**
 
-Run the below command to get the credential for the cluster **pinot-quickstart** that you just created or your existing cluster.&#x20;
+Run this command to get the credential for the cluster **pinot-quickstart** that you just created:
```

Comment:
```suggestion
Run the following command to get the credentials for the cluster **pinot-quickstart** that you just created:
```

Reply:
[following](https://developers.google.com/style/word-list#letter-f) is a bit more explicit that this